docs/content/src/learn/tutorials/messaging-integrations/reliable-delivery/src/message_processor/Module.md

```diff
@@ -1,171 +1,52 @@
-Implements the message processor for reliable delivery tutorial.
+Template for Message Processor Reliable Delivery
 
-# Reliable Message Delivery
+# Service Orchestration 
```

Comment:
Why are we calling this Service Orchestration?

Reply:
Fixed